### PR TITLE
Describe how to manage the sales reports

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request describes how to manage the sales reports

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

https://docs.magento.com/user-guide/reports/sales-reports.html

whatsnew
Updated the [Sales Reports](https://docs.magento.com/user-guide/reports/sales-reports.html) topic to include details about refresh, filtering, and export for sales reports.

### CODE REVIEW

File: src/reports/sales-reports.md [hunk 1]
```diff
@@ -59,3 +59,32 @@ The [PayPal Settlement Report]({% link payment/paypal-settlement-reports.md %})
...
+
+## How to manage Sales Reports
+
+### Step 1: Refresh the report statistics
+
+1. On the _Admin_ sidebar, go to **Reports**.
+
+1. Choose the reports you want to refresh.
+
+1. To update statistics, click on the anchor text `here`.
```
Comment (leslietilling) [inline src/reports/sales-reports.md:71]:
```suggestion
1. To update statistics, click **here** in the text.
```

File: src/reports/sales-reports.md [hunk 2]
```diff
@@ -59,3 +59,32 @@ The [PayPal Settlement Report]({% link payment/paypal-settlement-reports.md %})
...
+
+### Step 2: Filter reports
+
+Admin can get a whole website or one store report, a report for a time interval or a date. To filter sale report set:
```
Comment (leslietilling) [inline src/reports/sales-reports.md:75]:
```suggestion
In the Admin, you can access can generate a report for a whole website or for one store, within a time interval or a date. To filter a sale report, set the following options:
```

**Review by leslietilling — CHANGES_REQUESTED** [review]

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**